doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_keys.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
+
+.. _ug_nrf54h20_keys:
+
+Provisioning keys on the nRF54H20
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+###TO DO: add script link, expand description.
+
+The keys provisioning workflow for the nRF54H20 consists of two main steps:
+
+1. Generating the keys using a script provided with the |NCS|.
+#. Programming the keys to the nRF54H20 SoC.
+
+Generating the keys
+===================
+
+A script is used to generate the necessary cryptographic keys, BLOBs, and metadata required for provisioning.
+The script follows the PSA crypto standard to generate the required 28-byte key.
+
+The generated key data is stored in a JSON file, which serves as an input for the next step.
```

Comment:
Currently there is no script that actually generate this json file, I have made a version, but that script is not publicly available (yet).

Reply:
@evenl  Isn't this script in scope of RC2? The documentation assumes the script is there

Reply:
@shanthanordic @robertstypa FYI